config/settings/common.py

```diff
@@ -55,17 +55,30 @@
 # END APP CONFIGURATION
 
 # MIDDLEWARE CONFIGURATION
-MIDDLEWARE_CLASSES = (
-    # Make sure djangosecure.middleware.SecurityMiddleware is listed first
-    'djangosecure.middleware.SecurityMiddleware',
-    'django.contrib.sessions.middleware.SessionMiddleware',
+# https://docs.djangoproject.com/en/3.1/topics/http/middleware/
+# MIDDLEWARE = [
+#     # Make sure djangosecure.middleware.SecurityMiddleware is listed first
+#     'djangosecure.middleware.SecurityMiddleware',
+#     'django.contrib.sessions.middleware.SessionMiddleware',
+#     'corsheaders.middleware.CorsMiddleware',
+#     'django.middleware.common.CommonMiddleware',
+#     'django.middleware.csrf.CsrfViewMiddleware',
+#     'django.contrib.auth.middleware.AuthenticationMiddleware',
+#     'django.contrib.messages.middleware.MessageMiddleware',
+#     'django.middleware.clickjacking.XFrameOptionsMiddleware',
+# ]
+
+MIDDLEWARE = [
     'corsheaders.middleware.CorsMiddleware',
+    'django.middleware.security.SecurityMiddleware',
```

Comment:
Does `djangosecure.middleware.SecurityMiddleware` no longer need to be listed first?

Reply:
the order doesn't matter